langchain/document_loaders/parsers/html/bs4.py

```diff
@@ -0,0 +1,56 @@
+"""Loader that uses bs4 to load HTML files, enriching metadata with page title."""
+
+import logging
+from typing import Any, Dict, Iterator, Mapping, Optional, Union
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseBlobParser
+from langchain.document_loaders.blob_loaders import Blob
+
+logger = logging.getLogger(__name__)
+
+
+class BS4HTMLParser(BaseBlobParser):
+    """Parser that uses beautiful soup to parse HTML files."""
+
+    def __init__(
+        self,
+        features: str = "lxml",
```

Comment:
should we add this to bs4html loader as well?

Reply:
it's already there, just hidden in the init